telebot/types.py

```diff
@@ -1209,9 +1209,9 @@ def de_json(cls, json_string):
         if 'write_access_allowed' in obj:
             opts['write_access_allowed'] = WriteAccessAllowed.de_json(obj['write_access_allowed'])
             content_type = 'write_access_allowed'
-        if 'user_shared' in obj:
-            opts['user_shared'] = UserShared.de_json(obj['user_shared'])
-            content_type = 'user_shared'
+        if 'users_shared' in obj:
+            opts['users_shared'] = UserShared.de_json(obj['users_shared'])
+            content_type = 'users_shared' # COMPATIBILITY BROKEN!
```

Comment:
@Badiboy compatibility will be broken here. They renamed the property, and this makes me rename the content_type too. Any ideas?

Reply:
I don't see any better way... Let's hope not much people used this content type...